src/DefaultBuilder/src/WebHost.cs

```diff
@@ -215,6 +221,15 @@ internal static void ConfigureWebDefaults(IWebHostBuilder builder)
                     StaticWebAssetsLoader.UseStaticWebAssets(ctx.HostingEnvironment, ctx.Configuration);
                 }
             });
+            builder.ConfigureLogging(loggingBuilder =>
```

Comment:
I don't think this should be here. This should go into the generic host.

Reply:
This get's called by the generic host. I verified that when doing #22528.

Reply:
I thought we agreed to only to enable it for webapps?

Reply:
I think we should turn it on by default for all generic hosted apps. That means workers get it for free as well.

Reply:
So remove the `ConfigureWebDefaults` changes (make changes to generic host in the runtime repo) and keep the `WebHostBuilder` changes and get this merged?

Reply:
> I think we should turn it on by default for all generic hosted apps. That means workers get it for free as well.

Can we clarify the scope of this? I mean how common the apps are using default host against using none-defaulted hosts? I am asking because originally we decided to have the behavior as opt-in to be intentional paying the extra cost when only opting-in .

Reply:
Worker service and ASP.NET Core core applications.

Reply:
Created a new PR in runtime for this https://github.com/dotnet/runtime/pull/37892

Reply:
do we still need to have this PR?

Reply:
In theory this PR prevents a regression for existing WebHost customers.